e2e/tests/gatekeeperAdministersCaseAfterGatekeeping_test.js

```diff
@@ -99,13 +99,16 @@ Scenario('Gatekeeper enters hearing details and submits', async (I, caseViewPage
 
 Scenario('Gatekeeper drafts standard directions', async (I, caseViewPage, draftStandardDirectionsEventPage) => {
   await caseViewPage.goToNewActions(config.administrationActions.draftStandardDirections);
+  await draftStandardDirectionsEventPage.skipDateOfIssue();
   await draftStandardDirectionsEventPage.enterJudgeAndLegalAdvisor('Smith', 'Bob Ross');
   await draftStandardDirectionsEventPage.enterDatesForDirections(directions[0]);
   await draftStandardDirectionsEventPage.markAsDraft();
   await I.completeEvent('Save and continue');
   I.seeEventSubmissionConfirmation(config.administrationActions.draftStandardDirections);
+
   caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
-  I.see('draft-standard-directions-order.pdf');
+  I.seeSimpleAnswerInTab('Standard directions order', 'File','draft-standard-directions-order.pdf');
+  I.seeSimpleAnswerInTab('Standard directions order', 'Date of issue', dateFormat(new Date(), 'd mmmm yyyy'));
```

Comment:
It would probably be best to create the date just before the `draftStandardDirectionsEventPage.skipDateOfIssue()`. Given how long SDO generation can take it might introduce some additional flakiness if we create the date here.

Reply:
done

Reply:
done